tools/azure-rest-api-specs-examples-automation/directory/test_examples_dir.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+from examples_dir import try_find_resource_manager_example
+
+
+class TestExamplesDir(unittest.TestCase):
+
+    @unittest.skip(
+        'require call "_set_paths" with your local azure-rest-api-specs repository and azure-sdk-for-java repository'
+    )
+    def test_find_resource_manager_example_typespec(self):
+        example_dir = try_find_resource_manager_example(
+            "c:/github/azure-rest-api-specs",
+            "2024-03-01-preview",
+            "MongoClusters_ListConnectionStrings.json",
+            "c:/github/azure-sdk-for-java/sdk/mongocluster/azure-resourcemanager-mongocluster",
```

Comment:
Will these local paths cause the tests to not be able to run in CI?

Reply:
Currently this test is disabled for this reason.

Let me see if I can prepare a very simple local file structure during test run (in a temp folder) to make the test run in CI.